Assignment/solution_student/select_fun_part2/js/main.js

```diff
@@ -1,29 +1,49 @@
 // 1.	Slide the header up when it is clicked on.
-
+$('header').on('click', function(){
+	$('header').slideUp('slow');
+});
 // 2.	Remove the purplebox class
+$('*').removeClass('purplebox');
 
 // 3.	Hide all of the div’s in #row4
+$('#row4').hide();
 
 // 4.	Add a paragraph that says “I have been added” to all boxes with class “add-para” (Hint: you'll need to look up the append() method)
+$('.add-para').append('I have been added');
```

Comment:
Nice job! The only thing I would add is to wrap your text in a `<p>` tag, just so the text isn't laying out "naked" without a containing element, so something like:
```
$('.add-para').append('<p>I have been added</p>');
```